File: TransCoder_tokenized_test_set_functions/cpp/REPLACE_CHARACTER_C1_C2_C2_C1_STRING_S.cpp
string replace ( string s, char c1, char c2 ) {
  int l = s . length ( );
  for ( int i = 0;
  i < l;
  i ++ ) {
    if ( s [ i ] == c1 ) s [ i ] = c2;
    else if ( s [ i ] == c2 ) s [ i ] = c1;
  }
  return s;
}

File: TransCoder_tokenized_test_set_functions/java/REPLACE_CHARACTER_C1_C2_C2_C1_STRING_S.java
static String replace ( String s, char c1, char c2 ) {
  int l = s . length ( );
  char [ ] arr = s . toCharArray ( );
  for ( int i = 0;
  i < l;
  i ++ ) {
    if ( arr [ i ] == c1 ) arr [ i ] = c2;
    else if ( arr [ i ] == c2 ) arr [ i ] = c1;
  }
  return String . valueOf ( arr );
}